src/cs3panel.tsx

```diff
@@ -89,18 +91,61 @@ export class Cs3HeaderWidget extends ReactWidget {
     }
 }
 
+export const Bottom = (props: BottomProps): JSX.Element => {
+
+    const [text, setText] = useState(props.message);
+
+    const setLabel = async () => {
+        const showHidden: boolean = await props.db.fetch('showHidden') as boolean;
+        if (showHidden == undefined || !showHidden) {
+            const hiddenFilesNo: number = await props.db.fetch('hiddenFilesNo') as number;
+            setText('Hidden files ' + hiddenFilesNo + ' (show)')
+        } else {
+            setText('Cover hidden files')
+        }
+    }
+
+    props.browser.model.pathChanged.connect( async (browser, args) => {
+        await setLabel()
+    })
+
+    props.browser.model.refreshed.connect(async (browser) => {
+        await setLabel()
+    })
+
+    return (
+        <div className={'jp-bottom-div'} onClick={async () => {
+            await setLabel()
+        }}>
+            {text}
+        </div>
+    )
+}
+
 export class Cs3BottomWidget extends ReactWidget {
-    constructor(title: string, id: string, options: Widget.IOptions = {}) {
+    private bottomProps: { message: string; db: IStateDB; drive: CS3Contents, browser: FileBrowser };
```

Comment:
do we need message (see above) and drive?

Reply:
It seems we don't